geomstats/geometry/poincare_ball.py

```diff
@@ -10,7 +10,7 @@
 from geomstats.geometry.hyperbolic import Hyperbolic
 from geomstats.geometry.riemannian_metric import RiemannianMetric
 
-TOLERANCE = 1e-6
+TOLERANCE = gs.atol
```

Comment:
Maybe nicer to write gs.atol instead of TOLERANCE where it is used to avoid having a global variable (that won't be visible in the documentation whereas gs.atol should be)

Reply:
Done !